### PR TITLE
Add packaging. Reformat README.

### PR DESCRIPTION
**Pull request by bac**
Just discovered a project I work on uses `flowsynth`.  For our own purposes, it will be much easier if we can install it as a wheel.

One item of controversy: Will's email needs to be unobfuscated to be included in the packaging.

If this PR is accepted, it would be nice to have the package pushed to `pypi.org`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,30 @@
+
 # Flowsynth #
 
-Flowsynth is a tool for rapidly modelling network traffic. Flowsynth can be used to generate text-based hexdumps of packets as well as native libpcap format packet captures.
+Flowsynth is a tool for rapidly modeling network traffic. Flowsynth can be used
```
Comment (bac) [inline README.md]: Broke long lines and fixed typo s/modelling/modeling/.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+to generate text-based hexdumps of packets as well as native libpcap format
+packet captures.
 
 ## Installation ##
 
-The following python modules are required to run Flowsynth:
+`pip install .`
```
Comment (bac) [inline README.md]: Added new installation instruction.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
+
+If you want to run the source locally without installing, he following python
+modules are required to run Flowsynth:
```
Comment (bac) [inline README.md]: All remaining changes to this file were just to break long lines.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    'scapy>=2.4.0',
+    'argparse',
+]
+
+print find_packages()
+
+if __name__ == '__main__':
+    setup(
+        name='flowsynth',
+        version='0.1',
```
Comment (bac) [inline setup.py]: Some versioning system should be used here.

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        author='Will Urbanski',
+        author_email='will.urbanski@gmail.com',
+        install_requires=install_requires,
+        license='Apache License 2.0',
+        url='https://github.com/secureworks/flowsynth',
+        packages=find_packages(),
+        scripts=[
+            'flowsynth/flowsynth.py',
+        ],
+        classifiers=[
```
Comment (bac) [inline setup.py]: These classifiers were just made up and need to be reviewed for accuracy.

File: README.md (hunk 1 above)
Comment (counterthreatunit) [reply README.md]: We have been here before with the alleged "typo" ... See https://github.com/secureworks/flowsynth/pull/5.   Personally, I prefer the American style.

File: README.md [hunk 6]
```diff
@@ -1,19 +1,30 @@
...
+Flowsynth is a tool for rapidly modeling network traffic. Flowsynth can be used
+to generate text-based hexdumps of packets as well as native libpcap format
+packet captures.
 
 ## Installation ##
 
-The following python modules are required to run Flowsynth:
+`pip install .`
+
+If you want to run the source locally without installing, he following python
```
Comment (counterthreatunit) [inline README.md]: s/ he following/the following/

File: README.md (hunk 1 above)
Comment (bac) [reply README.md]: Haha, I can live with reverting that one.

Comment (bac) [reply README.md]: Then there is this find:
https://github.com/secureworks/flowsynth/blob/49ba2c172812b110eb8af2ab778e4baecfedc15b/src/flowsynth.py#L3

**Comment by bac** [comment]
@counterthreatunit Would you please take a look at the content of this proposal and act on it?  Thanks.

**Comment by counterthreatunit** [comment]
Implemented in commit https://github.com/secureworks/flowsynth/commit/950b8d0d23adde43e98332407766efa1523908c5.

**Pull request closed without merging**